### PR TITLE
ci: Do not build the web-app-reporter for `funTest-docker`

### PR DESCRIPTION
**Pull request by mnonnenmacher**
WIP: Test if not building the web app reporter fixes the fun tests.

Test if not building the web-app-reporter for

Please ensure that your pull request adheres to our [contribution guidelines](https://github.com/oss-review-toolkit/.github/blob/main/CONTRIBUTING.md). Thank you!

### CODE REVIEW

File: batect.yml [hunk 1]
```diff
@@ -72,7 +72,7 @@ tasks:
       A convenience task to build (but not run) ORT's functional tests.
     run:
       container: build
-      command: ./gradlew --stacktrace jar funTestClasses
+      command: ./gradlew --stacktrace -x :reporter-web-app:yarnBuild jar funTestClasses
```
Comment (sschuberth) [inline batect.yml:75]: This needs to say `-x :plugins:reporters:web-app-template:yarnBuild` now. Same below.

Comment (mnonnenmacher) [reply batect.yml:75]: Right, copy paste error...

**Comment by mnonnenmacher** [comment]
> This doesn't seem to help sob

It's weird that we don't get the logs anymore. Past runs had this out of disk space message, but now there is just a failed job without logs.

**Comment by sschuberth** [comment]
> It's weird that we don't get the logs anymore. Past runs had this out of disk space message, but now there is just a failed job without logs.

In exchange, I've seen the job run properly (but failing due to regular test flakyness) recently again...

**Comment by mnonnenmacher** [comment]
Issue was fixed by #7110.

**Pull request closed without merging**